docs/marketplace-and-submit-job-v2/marketplace-and-submit-job-v2.md

```diff
@@ -79,40 +79,326 @@ And you can share the script, data, docker images or whole job configuration on
+| :--------------------------------- | :--------------------------------------- |
+| PAI_WORK_DIR                       | Working directory in Docker container    |
+| PAI_DEFAULT_FS_URI                 | Default file system uri in PAI           |
+| PAI_JOB_NAME                       | `job.name` in config file                 |
+| PAI_JOB_VC_NAME                    | The virtual cluster in which the job is running     |
+| PAI_USER_NAME                      | User who submit the job                  |
+| PAI_CURRENT_TASK_ROLE_NAME         | `task.role` of current task role     |
+| PAI_CURRENT_TASK_ROLE_TASK_COUNT   | `task.resource.instances` of current task role |
+| PAI_CURRENT_TASK_ROLE_CPU_COUNT    | `task.resource.cpu` of current task role  |
+| PAI_CURRENT_TASK_ROLE_MEM_MB       | `task.resource.memoryMB` of current task role   |
+| PAI_CURRENT_TASK_ROLE_SHM_MB       | `task.resource.shmMB` of current task role      |
+| PAI_CURRENT_TASK_ROLE_GPU_COUNT    | `task.resource.gpu` of current task role  |
+| PAI_CURRENT_TASK_ROLE_MIN_FAILED_TASK_COUNT    | `task.minFailedTaskCount` of current task role    |
+| PAI_CURRENT_TASK_ROLE_MIN_SUCCEEDED_TASK_COUNT | `task.minSucceededTaskCount` of current task role |
+| PAI_CURRENT_TASK_ROLE_CURRENT_TASK_INDEX | Index of current task in current task role, starting from 0 |
+| PAI_JOB_TASK_COUNT                 | Total tasks' number in config file       |
+| PAI_JOB_TASK_ROLE_COUNT            | Total task roles' number in config file  |
+| PAI_JOB_TASK_ROLE_LIST             | Comma separated all task role names in config file |
+| PAI_CONTAINER_HOST_IP              | Allocated ip for current docker container |
+| PAI_CONTAINER_HOST_PORT_LIST       | Allocated port list for current docker container, in `portLabel0:port0,port1,port2;portLabel1:port3,port4` format |
+| PAI_CONTAINER_HOST\_`$type`\_PORT_LIST | Allocated port list for `portList.label == $type`, comma separated `port` string |
+| PAI_TASK_ROLE\_`$name`\_HOST_LIST  | Host list for `TASK_ROLE_NAME == $name`, comma separated `ip:port` string, sorted by current task index in task role. Each task role has a host list environment variable with the corresponding task role name |
+
+Below is an example for a distributed tensorflow image classification training job :
 
 ```yaml
 protocol_version: v2
-name : tensorflow_serving_mnist
-type : job
-version : 1.0.0
-contributor : Qi chen
-description : image classification, mnist dataset, tensorflow, serving
-retryCount: -2
-
-tasks :
-  - role : worker
-    dockerimage : tf_serving_example
+name: tensorflow_cifar10              # name of the job
+type: job
+version: 1.0.0
+contributor: Alice                    # contributor of the job
+description: image classification, cifar10 dataset, tensorflow, distributed training
+retryCount: 0                         # job retry count
+virtualCluster: default               # the virtual cluster job runs on
+
+parameters:
+  model: resnet20
+  batchsize: 32
```

Comment:
missing `data` parameter?

Reply:
data and script are optional. If user create his dockerimage and put data and script directly into it, he can directly run the command without specify the data and script.